CHANGELOG.md

```diff
@@ -2,6 +2,112 @@ TODO: This document was manually maintained so might be incomplete. The
+
+    * [https://github.com/kubernetes/kubernetes/pull/53743](https://github.com/kubernetes/kubernetes/pull/53743)
+
+* `meta.MetadataAccessor` got API chunking support
+
+    * [https://github.com/kubernetes/kubernetes/pull/53768](https://github.com/kubernetes/kubernetes/pull/53768)
+
+* `core/v1.DefaultHardPodAffinitySymmetricWeight` type changed from int to int32
+
+    * [https://github.com/kubernetes/kubernetes/pull/53850](https://github.com/kubernetes/kubernetes/pull/53850)
+
+* Renamed `core/v1.ScaleIOVolumeSource` to `ScaleIOPersistentVolumeSource`
+
+    * [https://github.com/kubernetes/kubernetes/pull/54013](https://github.com/kubernetes/kubernetes/pull/54013)
+
+* Renamed `core/v1.RBDVolumeSource` to `RBDPersistentVolumeSource`
+
+    * [https://github.com/kubernetes/kubernetes/pull/54302](https://github.com/kubernetes/kubernetes/pull/54302)
+
+* Removed `core/v1.CreatedByAnnotation`
+
+    * [https://github.com/kubernetes/kubernetes/pull/54445](https://github.com/kubernetes/kubernetes/pull/54445)
+
+* Added `storage.k8s.io/v1alpha1` API group
+
+    * [https://github.com/kubernetes/kubernetes/pull/54463](https://github.com/kubernetes/kubernetes/pull/54463)
+
+* Informers got a NewFilteredSharedInformerFactory to e.g. filter by namespace
+
+    * [https://github.com/kubernetes/kubernetes/pull/54660](https://github.com/kubernetes/kubernetes/pull/54660)
+
+* Renamed `core/v1.StorageMediumHugepages` to `StorageMediumHugePages`
+
+    * [https://github.com/kubernetes/kubernetes/pull/54748](https://github.com/kubernetes/kubernetes/pull/54748)
+
+* Renamed `core/v1.StorageMediumHugepages` to `StorageMediumHugePages`
+
+    * [https://github.com/kubernetes/kubernetes/pull/54748](https://github.com/kubernetes/kubernetes/pull/54748)
+
+* Do not truncate body in glog output with log level 10
+
+    * [https://github.com/kubernetes/kubernetes/pull/54801](https://github.com/kubernetes/kubernetes/pull/54801)
+
+* `+groupName`tag added to allow unique Go identifiers in clientsets and informers
+
+    * [https://github.com/kubernetes/kubernetes/pull/54950](https://github.com/kubernetes/kubernetes/pull/54950)
+
+* Added support for conditions in StatefulSet status
+
+    * [https://github.com/kubernetes/kubernetes/pull/55268](https://github.com/kubernetes/kubernetes/pull/55268)
+
+* Added support for conditions in DaemonSet status
+
+    * [https://github.com/kubernetes/kubernetes/pull/55272](https://github.com/kubernetes/kubernetes/pull/55272)
+
+* The dynamic admission webhook is split into two kinds, mutating and validating.
+
+    * [https://github.com/kubernetes/kubernetes/pull/55282](https://github.com/kubernetes/kubernetes/pull/55282)
+
+* If you upgrade your client-go libs and use the `AppsV1()` interface, please note that the default garbage collection behavior is changed.
+
+    * [https://github.com/kubernetes/kubernetes/pull/55148](https://github.com/kubernetes/kubernetes/pull/55148)
+
```

Comment:
Can we add an action-required notice in the changelog saying that clientset's default versioned group interface (like https://github.com/kubernetes/kubernetes/blob/f875ee596d4eb0d5847cf2f0e6b13a80de7d6008/staging/src/k8s.io/client-go/kubernetes/clientset.go#L58-L59) will be removed in next release.

Reply:
Should this go as a note for the next release or should it be included for this one? Since the removal will be done for the next one.

Makes sense to have it here as well, but just wanted to double check.

Reply:
You're right. Adding it as a note for the next release is better.